### PR TITLE
Add portstat collector

### PR DESCRIPTION
**Pull request by Nihn**
@mwicat is contributor of this collector.

### CODE REVIEW

File: src/collectors/portstat/portstat.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+from collections import Counter
+
+import psutil
+import diamond.collector
+
+
+def get_port_stats(port):
+    '''
+    Iterate over connections and counts states for specified port
```
Comment (deejay1) [inline src/collectors/portstat/portstat.py]: language: "*Iterates* over connections" or "Iterate over connections and *count*"

File: src/collectors/portstat/portstat.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    :param port: port which stats are collected
```
Comment (deejay1) [inline src/collectors/portstat/portstat.py]: "port *for which* stats are collected"

File: src/collectors/portstat/portstat.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+"""
+
+from collections import Counter
+
+import psutil
+import diamond.collector
+
+
+def get_port_stats(port):
```
Comment (deejay1) [inline src/collectors/portstat/portstat.py]: Wouldn't it be cleaner to "hide" that inside the collector class? Just asking ;)

**Comment by Nihn** [comment]
ping :).

**Comment by josegonzalez** [comment]
@deejay1 any objections to merging?

**Comment by deejay1** [comment]
@josegonzalez :+1:

**Pull request merged**